prometheus/README.md

```diff
@@ -41,6 +41,8 @@ Due to the nature of this integration, it's possible to submit a high number of
 
 All metrics collected by the prometheus check are forwarded to Datadog as custom metrics.
 
+The metric `send_monotonic_counter` defaults to `True` and uses `self.monotonic_count`, so the agent sends the deltas and the in-app type is count. With `send_monotonic_counter: False` Datadog sends the raw, monotonically increasing value, and the in-app type is gauge.
```

Comment:
`send_monotonic_counter` is a configuration variable not a metric. If set to true the **agent** uses `self.monotonic_count`

I feel like this part should be in the Configuration section

Reply:
This is a tricky thing to be clear about. What if we just leave out `self.monotonic_count` entirely? Like, is it necessary to understand the behaviour? (open question)

```suggestion
If `send_monotonic_counter: True`, the Agent sends the deltas of the values in question, and the in-app type is set to count (this is the default behaviour). If `send_monotonic_counter: False`, the Agent sends the raw, monotonically increasing value, and the in-app type is set to gauge.
```

Agreed that it should be in the configuration section.

Reply:
okay - accepted @phrawzty suggestion and I moved to configuration section